recipe/meta.yaml

```diff
@@ -41,25 +30,35 @@ source:
     - patches/handle-abs-compiler-path.patch
     # Fix little problems with the test suite.
     - patches/fix-flaky-tests.patch
-    # Fixes usage of non-standard ssize_t.
-    # - patches/remove-non-standard-ssize_t.patch  # [win] #upstreamed
     # MAP_ANONYMOUS not defined
     - patches/fix_map_anonymous.patch
     - patches/cmake-find-ideep-mkldnn-in-sourcetree.patch # [x86] #required for mkl implementation
     - patches/suppress_Wbitwise_instead_of_logical_clang.patch
+    # This is required in order to use our Pybind instead of the one
+    # submoduled into the Pytorch repo
+    - patches/cmake-fix-pybind-option.patch
+    # This patch was taken from conda-forge. There was a variable
+    # which directed the build system to use mkldnn for caffe2
+    # specifically, but since it was just turned on by the global
+    # mkldnn variable anyway, it was superfluous.
+    # Some of the changes in the original patch had been upstreamed.
+    - patches/cmake-unify-USE_MKLDNN-and-CAFFE2_USE_MKLDNN.patch
```

Comment:
please make sure that patches are treated always with linux line-endings.  See here .gitattributes file and adjust rules for *.diff and *.patch as needed

Reply:
thanks, they are